src/main/webapp/app/modules/provider/all-records.tsx

```diff
@@ -290,52 +364,7 @@ export class AllRecords extends React.Component<IAllRecordsProps, IAllRecordsSta
             <ButtonPill onClick={this.toggleFilter} translate="providerSite.filter" />
           </div>
         </div>
-        {isMapView ? (
-          <Row className="mb-4 mx-3">
-            <Col md={isRecordHighlighted || filterOpened ? 8 : 12} className="pb-2 pl-0 pr-1 map-view">
-              <Map
-                googleMapURL={mapUrl}
-                records={allRecords}
-                lat={selectedLat}
-                lng={selectedLng}
-                loadingElement={<div style={{ height: '100%' }} />}
-                containerElement={<div style={{ height: '400px' }} />}
-                mapElement={<div style={{ height: '100%' }} />}
-                onMarkerClick={this.selectRecord}
-              />
-            </Col>
-            {isRecordHighlighted && selected && !filterOpened ? (
-              <Col md={4} className={`col-md-4 pr-0 selected-record`}>
-                <RecordCard record={selected} link={`single-record-view/${selected.organization.id}`} />
-              </Col>
-            ) : null}
-            <MediaQuery minDeviceWidth={769}>
-              {filterOpened &&
-                !isRecordHighlighted && (
-                  <Col md={4}>
-                    <div className="filter-card mx-3 mb-4">
-                      <FilterCard dropdownOpen={filterOpened} toggleFilter={this.toggleFilter} getFirstPage={this.getFirstPage} />
-                    </div>
-                  </Col>
-                )}
-            </MediaQuery>
-          </Row>
-        ) : (
-          <div>
-            <Row noGutters>
-              <MediaQuery maxDeviceWidth={768}>{this.mapRecords(allRecords)}</MediaQuery>
-              <MediaQuery minDeviceWidth={769}>{filterOpened ? this.mapWithFilter(allRecords) : this.mapRecords(allRecords)}</MediaQuery>
-            </Row>
-            <div className="button-pill mb-4 text-center">
-              <div
-                className={`d-inline button-pill ${hasReachedMaxItems ? 'disabled' : ''}`}
-                onClick={() => this.handleLoadMore(hasReachedMaxItems)}
-              >
-                <Translate contentKey="providerSite.loadMore" />
-              </div>
-            </div>
-          </div>
-        )}
+        {isMapView ? <this.mapView /> : <this.gridView />}
```

Comment:
Why these functions are in angle brackets?

Reply:
This is what I've recently learned. In my opinion it looks more like component than this.mapView().

Reply:
If eslint is fine with that, then ok